tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +72,23 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// setCacheSizeLimits sets the content cache size limits for an existing repository
+// the repository server is connected to.
+func (mcs *MultiClientSnapshotter) setContentCacheSizeLimit(contentCacheSize int) error {
+	_, _, err := mcs.server.Run("cache", "set",
+		contentCacheLimitMBFlag, strconv.Itoa(contentCacheSize))
+
+	return err
+}
+
+// setMetadataCacheSizeLimit sets the metadata cache size limits for an existing repository, the repository server is connected to.
+func (mcs *MultiClientSnapshotter) setMetadataCacheSizeLimit(metadataCacheSize int) error {
+	_, _, err := mcs.server.Run("cache", "set",
+		metadataCacheLimitMBFlag, strconv.Itoa(metadataCacheSize))
+
+	return err
+}
```

Comment:
Actually one more quick question: it looks like both flags can be passed simultaneously to the `cache set` call, is that right? Perhaps we can simplify this even more by merging these two functions into `setCacheSizeLimit(content, metadata int) error`? If a non-negative integer is passed for one or both arguments, we can add the appropriate flag(s) and only need to call a single `cache set` command. Thoughts?

Reply:
yes, both flags can be passed simultaneously according to https://kopia.io/docs/reference/command-line/advanced/cache-set/
So, just one command that sets both the cache size limits by default? One could argue, two separate functions may promote reuse later as compared to a single function.

Reply:
I think there's some personal preference here; either way probably works fine. The proposal would accommodate setting one or both, e.g. `setCacheSizeLimit(100, -1)` would only set the content limit to 100 MB (hence the "non-negative" choice of words above). But some might argue that's less readable as well, and sometimes minor performance optimizations can be neglected in favor of better readability. Proceed as desired, my prior approval still stands!